client/src/i18n/index.js

```diff
@@ -164,6 +164,141 @@ const factory = (options = { debug: true }) =>
+            share: {
+              copied: 'Skopiowano!',
+              tooltip: 'pokaż kod qr',
+              allowEdit: 'zezwól na edycję',
+            },
+            sessionDetails: {
+              name: 'Nazwa Twojej sesji',
+              tts: 'TTS',
+              split: 'Split',
+              startDate: 'Data rozpoczęcia sesji',
+              endDate: 'Data zakończenia sesji',
+              duration: 'Jak długo trwała sesja? (roughly)',
+              durationUnit: 'godziny',
+              detailsSavedCorrectly: 'Detale zapisane pomyślnie',
+              vpChangeConfirmation: {
+                title: 'Zmieniłeś ilość punktów zwycięstwa',
+                content:
+                  'Zmieniłeś ilość punktów zwycięstwa.\nNiektórzy gracze zdążyli zdobyć punkwy zwycięstwa, co oznacza, że gra jest w trakcie.\nJesteś pewny, że chcesz zmienić ilość punktów zwycięstwa?',
+              },
+            },
+            fullscreen: {
+              tooltip: 'pokaż w trybie pełnoekranowym',
+            },
+            shuffle: {
+              tooltip: 'pomieszaj kolejność ras',
+              shuffled: 'rasy pomieszane',
+            },
+            publicObjectives: {
+              labels: {
+                add: 'dodaj cel',
+                new: 'nowy cel',
+              },
+            },
+            sessionList: {
+              title: 'Twoje zapamiętane sesje',
+              fullAccess: 'Pełen dostęp',
+              new: 'Nowa Sesja',
+              secondaryTitle: '(rasy: {{factionList}})',
+            },
+            kb: {
+              title: 'Baza wiedzy',
+              panels: {
+                sI: {
+                  button: 'Przeglądaj cele fazy I',
+                  title: 'Cele fazy I',
+                },
+                sII: {
+                  button: 'Przeglądaj cele fazy II',
+                  title: 'Cele fazy II',
+                },
+                secretObj: {
+                  button: 'Przeglądaj sekretne cele',
+                  title: 'Sekretne cele',
+                },
+                exploration: {
+                  button: 'Przeglądaj karty eksploracji {{type}}',
+                  title: 'karty eksploracji {{type}}',
+                  types: {
+                    cultural: 'kulturowe',
+                    hazardous: 'niegościnne',
+                    industrial: 'przemysłowe',
+                    frontier: 'pogranicza',
+                  },
+                },
+                relics: {
+                  button: 'Przeglądaj relikty',
+                  title: 'Relikty',
+                },
+                strategy: {
+                  button: 'Przeglądaj karty strategii',
+                  title: 'Karty strategii',
+                },
+              },
+            },
+            deletePO: {
+              tooltip: 'Usuń cel publiczny',
+              content:
+                'Ten cel publiczny został spełniony przez gracza. Jesteś pewny, że chcesz go usunąć?',
```

Comment:
```suggestion
                'Ten cel publiczny został spełniony przez gracza. Czy na pewno chcesz go usunąć?',
```
let's use non-gender-specific forms